pkgs/by-name/co/composefs/package.nix

```diff
@@ -45,27 +50,45 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.enableFeature enableValgrindCheck "valgrind-test")
   ];
 
-  nativeBuildInputs = [ autoreconfHook go-md2man pkg-config ];
-  buildInputs = [ openssl ]
+  nativeBuildInputs = [
+    autoreconfHook
+    go-md2man
+    pkg-config
+  ];
+  buildInputs =
+    [ openssl ]
     ++ lib.optional fuseSupport fuse3
-    ++ lib.filter (lib.meta.availableOn stdenv.hostPlatform) (
-    [
+    ++ lib.filter (lib.meta.availableOn stdenv.hostPlatform) ([
       libcap
       libseccomp
-    ]
-  );
+    ]);
 
   doCheck = true;
-  nativeCheckInputs = [ python3 which ]
+  nativeCheckInputs =
+    [
+      python3
+      which
+    ]
     ++ lib.optional enableValgrindCheck valgrind
     ++ lib.optional fuseSupport fuse3
-    ++ lib.filter (lib.meta.availableOn stdenv.buildPlatform) [ erofs-utils fsverity-utils ];
+    ++ lib.filter (lib.meta.availableOn stdenv.buildPlatform) [
+      erofs-utils
+      fsverity-utils
+    ];
 
   preCheck = ''
     patchShebangs --build tests/*dir tests/*.sh
     substituteInPlace tests/*.sh \
       --replace " /tmp" " $TMPDIR" \
       --replace " /var/tmp" " $TMPDIR"
+    substituteInPlace tests/test-units.sh \
+      --replace 'BINDIR="$1"' 'BINDIR=$(realpath "$1")' \
+      --replace 'composefs-info' '$BINDIR/composefs-info'
+  '';
+
+  postFixup = lib.optionalString (!installExperimentalTools) ''
+    rm $out/share/man/man5/composefs-dump.*
+    rmdir --ignore-fail-on-non-empty $out/share/man/man5
```

Comment:
```suggestion
    rm -r $out/share/man/man5
```

Reply:
This code is actually written so it won't delete the folder if they add another man.5 page in the future. Which this version would do.

Reply:
Turns out the naming is just a coincidence and the man5 is about additional functionality for `composefs-info`, so I've removed this code.